### ISSUE TITLE
Reorganize externalNORSERun.py sections into functions

### ISSUE THREAD

**Issue by olaszsoma** [osrep/NORSE_actor#8]
Many parts of externalNORSERun.py[](https://github.com/osrep/NORSE_actor/blob/development/externalNORSERun.py) are coherent sections, which could be moved to separate functions altogether. This reorganization has to be done before release.

**Comment by olaszsoma** [comment]
The interface will be organized into an object, which will call the Matlab NORSE program. This will have to be done later.

**Comment by olaszsoma** [comment]
This issue will be postponed, it will be removed from NORSE release 1.0.0.
